transform/models/marts/repo_pushes_daily.sql

```diff
@@ -0,0 +1,9 @@
+SELECT
+    repo_id,
+    repo_name,
+    DATE_TRUNC('day', event_date) AS date_day,
+    COUNT(*) AS push_count,
+    SUM(COUNT(*)) OVER (PARTITION BY repo_id ORDER BY DATE_TRUNC('day', event_date) ASC) AS cumul_push_count
```

Comment:
Minor nit picks: You can use `sum(push_count)` to avoid repeating the `count(*)`.

Reply:
Similar minor nit: You can use `ORDER BY date_day` to avoid repeating the `DATE_TRUNC` expression.